src/app/results/results.component.css

```diff
@@ -365,41 +365,141 @@ div.autocorrect{
   margin-left: 6%;
 }
 
-@media screen and (min-width:1920px) {
-  #tools {
-    margin-left: 27%;
+
+@media screen and (max-width: 1920px) {
+  #search-options {
+    margin-left: 5.5%
+  }
+  div.result{
+    margin-left: -5.7%;
+  }
+  div.autocorrect{
+    margin-left: 0;
+  }
+  div.result.message-bar{
+    margin-left: 0;
+  }
+  div.feed.col-md-10.col-md-offset-1.col-sm-10.col-sm-offset-1.col-lg-10.col-lg-offset-1 {
+    width: 34%;
   }
 }
 
-@media screen and (max-width: 1440px) {
-  .feed {
-    width: 43%;
+@media screen and (max-width: 1800px) {
+  #search-options {
+    margin-left: 6.5%
+  }
+  div.result{
+    margin-left: -2.7%;
+  }
+  div.autocorrect{
+    margin-left: 1%;
+  }
+  div.result.message-bar{
+    margin-left: 1%;
+  }
+  div.feed.col-md-10.col-md-offset-1.col-sm-10.col-sm-offset-1.col-lg-10.col-lg-offset-1 {
+    width: 36%;
   }
 }
+@media screen and (max-width: 1750px){
```

Comment:
why are elements changing the dimensions for some arbitrary values?

changing for some 25px ,100 px,  or 150px is not we have to do.
see this https://css-tricks.com/optimizing-large-scale-displays/ ,he didn't use media tags for each 50px .
**what could be an alternative?**
correct me if I am  wrong, I think the main reason you are using for each 50 px is because if you want to align element A with same width as element B, what you are doing is: hardcoding element A in every 50px dimention such that element A match the alignment with element B.
so the actual error is with element B, the alignment it follows is not same in all dimentions and due to which you have to change the dimensions required.
Ok why is the dimension of element B changing for arbitrary screen sizes?
![image](https://user-images.githubusercontent.com/15216503/28557024-bd8f56da-7127-11e7-92b0-a247b738c7f0.png)

look at the example above:
**problem we need to solve:**
align violet box below yellow box
**present situation:**
blue box has margins and paddings changing at screen sizes: 480, 281,584,530
red box has dimensions changing at screen sizes: ,1107,823,767,635,435,479
so now to align violet box with yellow box you have to change the dimensions of violet at  480, 281,584,530,1107,823,767,635,435,479 but since we cannot identify exact screen sizes the dimensions change for different elements, what we are doing is we are writing css for every 50px.
**solution:**
have only fixed screen sizes for all css files, I recommend using the same as CSS tricks used.so that you will now have an idea of at which screen size other elements are changing their dimensions.I would also recommend using some fluid grids

also please try following mathematics don't go by just trying different values that match the alignment.for instance.
![image](https://user-images.githubusercontent.com/15216503/28557530-9762b4c2-712a-11e7-9fa1-2018aa41d298.png)

in the above picture to align violet box with yellow box try using A+B as it's margin-left.
the above examples are just how the present